jboss/container/java/certs/bash/artifacts/trust-ose-cert.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+if [ -z "$JAVA_DISABLE_TRUST_OPENSHIFT_CA" ]; then
```

Comment:
note that there are 2 different CA...  the ca.crt , which I would call "the openshfit CA",  and the service-ca.crt is a very specific one  (https://docs.openshift.com/online/dev_guide/secrets.html#service-serving-certificate-secrets ) 

I'd prefer if we (optionally) added both.. However, even if we don't, we should make it clear in the names of the env and comments which CA is being (or not being) added.

Reply:
agreed